tests/phpunit/Admin/SettingsTest.php

```
@@ -0,0 +1,250 @@
+	public function tearDown(): void {
+		parent::tearDown();
+		// Reset mocks
+		self::$mock_valid_post_types_filter = null;
+		self::$mock_wpdb_get_var_result = 0;
+
+		// Restore original edac_post_types option
+		if (false === $this->original_edac_post_types) {
+			delete_option('edac_post_types');
+		} else {
+			update_option('edac_post_types', $this->original_edac_post_types);
+		}
```

PHPMD (2.15.0)

The method tearDown uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)